docs/technical-guide.md

```diff
@@ -0,0 +1,44 @@
+# Technical Guide
+
+Start with [Key Concepts](/docs/key-concepts.md) for an overview of concepts/terms.
+
+## Data model
+
+You can also explore the data model on the [Mermaid data model diagram](/docs/data-model.mmd).
```

Comment:
I think this will be best done in a GitHub link as well.

Or we could use mkdocs mermaid and stick this into a markdown file instead. Easiest to link to GitHub.

Reply:
Have used an absolute link to Github.